addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -26,17 +26,14 @@
                         <a t-if="phone" class="o-mail-avatar-card-tel text-truncate" t-att-title="phone" t-att-href="'tel:'+phone">
                             <i class="fa fa-fw fa-phone me-1"/><t t-esc="phone"/>
                         </a>
-                    </div>
-                </div>
-                <div class="flex-wrap gap-2 mt-2">
-                    <div class="justify-content-end d-flex align-items-baseline">
-                        <div class="d-flex gap-2 o_avatar_card_buttons">
+                        <div class="o_avatar_card_buttons d-flex gap-2 mt-2">
                             <button t-if="!user.share" class="btn btn-secondary btn-sm" t-on-click.stop="onSendClick">Send message</button>
                             <button t-if="showViewProfileBtn" class="btn btn-secondary btn-sm" t-custom-click.stop="(ev, isMiddleClick) => this.onClickViewProfile(isMiddleClick)" t-ref="viewProfileBtn">View Profile</button>
                         </div>
                     </div>
                 </div>
             </div>
+            <div t-if="skills?.length or roles?.length" class="o_avatar_card_details card-footer d-flex flex-column gap-3 rounded-bottom bg-100"/>
```

Comment:
Hello, it would be best to create a gedder "hasFooter" or something like that, that is overridden in other modules depending on their needs, rather than referencing non-existing fields in mail.

Reply:
Thanks @seb-odoo, indeed, I just realised I introduced these here, indeed 😅 

That being said, creating a Getter is a bit out of my area of expertise, so I tried to achieve it with QWeb but I guess I can't alterate the parent template when I'm in hr_skills/planning 😅 

I know this isn't related to your module but would you mind providing some guidance here or indicating the right direction?

Thanks 🙏

Reply:
Add a new get ...() in AvatarCardPopover component (.js file). Then patch the component in other modules.

You can look at `fieldNames`, it's patched in `hr` to add extra file.

Reply:
Note: you could technically patch the template directly but it often gets messy, especially when 2 different patches have to independently patch a specific part such as a `t-if`.

Patching template is usually best kept for actually adding nodes, but not so much for specific conditions/configuration.

Reply:
I'll have a look, thanks 🙏